fish_prompt.fish

```diff
@@ -391,11 +389,12 @@ function __bobthefish_prompt_virtualfish -d "Display activated virtual environme
   set_color normal
 end
 
-function __bobthefish_prompt_rubies -d 'Display current Ruby (rvm/rbenv)'
-  [ "$theme_display_ruby" = 'no' ]; and return
-  if which rvm-prompt >/dev/null 2>&1
-    set ruby_version (rvm-prompt i v g)
-  else if which rbenv >/dev/null 2>&1
+function __bobthefish_show_ruby -d 'Current Ruby (rvm/rbenv)'
+  set -l ruby_version
+  set -q theme_display_ruby_format; or set -l theme_display_ruby_format i v g
+  if type rvm-prompt >/dev/null 2>&1
```

Comment:
Changing these back to `type` from `which` is a regression (likely from a merge conflict resolution). Please keep the `which` version :)